chpl/chpl-service/src/main/java/gov/healthit/chpl/builder/ComplaintsWorksheetBuilder.java

```diff
@@ -54,7 +59,7 @@
     private static final int COL_CRITERIA_ID = 8;
     private static final int COL_CHPL_ID = 9;
     private static final int COL_SURV_ID = 10;
-    //columns 11 - 14 get hidden
+    //columns L-O get hidden
```

Comment:
Would it make sense to go back to 11 - 14, set them as constants, and use them in the for loop a few sections down? Or maybe have an array of "hidden" booleans that we could use to flip column hiding stuff easily as needed? I'm thinking about that ONC review column thing...